source/_layouts/has_complimentary.liquid

```diff
@@ -0,0 +1,27 @@
+---
+layout: base
+---
+
+<div class="sl-l-large-holy-grail">
+    <div class="sl-l-large-holy-grail__body">
+        <div class="sl-l-large-holy-grail__main">
+            <div class="sl-l-container sl-l-container--small">
+                <div class="sl-c-introduction">
+                    {{ introduction}}
+                </div>
+                
+                {{ content | typogr  }}
+            </div>
+        </div>
+            
+        {% if complementary_content %}
+        <div class="sl-l-large-holy-grail__complementary">
+                {% if complementary_content_nav %}<nav aria-label={{ complementary_content_aria_label }} class="sl-c-list-navigation-wrapper">{% endif %}
```

Comment:
@jgerigmeyer @mirisuzanne curious if y'all know a better way to optionally wrap this "complementary" content in `nav` tags. This way works but feels a bit strange.

Reply:
This is part of the same question I raised here: https://github.com/oddbird/sass-site/pull/6#discussion_r1088406943